selfdrive/car/interfaces.py

```diff
@@ -250,8 +250,8 @@ def create_common_events(self, cs_out, extra_gears=None, pcm_enable=True, allow_
       # Enable OP long on falling edge of enable buttons (defaults to accelCruise and decelCruise, overridable per-port)
       if not self.CP.pcmCruise and (b.type in enable_buttons and not b.pressed):
         events.add(EventName.buttonEnable)
-      # Disable on rising edge of cancel for both stock and OP long
-      if b.type == ButtonType.cancel and b.pressed:
+      # Disable on rising and falling edge of cancel for both stock and OP long
+      if b.type == ButtonType.cancel:
```

Comment:
this is to prevent a fault on GM where you can go from cancel to resume (which enables on resume rising edge now now). Cancel still has precedence. Weirdly, this causes no fault at a standstill, only while moving